ste/mgr-JobPartTransferMgr.go

```diff
@@ -272,6 +272,20 @@ func (jptm *jobPartTransferMgr) Info() TransferInfo {
 		src = sUrl.String()
 	}
 
+	if versionID != "" {
+		versionID = "versionId=" + versionID
+		sURL, e := url.Parse(src)
+		if e != nil {
+			panic(e)
+		}
+		if len(sURL.RawQuery) > 0 {
+			sURL.RawQuery += "&" + versionID
```

Comment:
Minor: Use query.Add() instead. As in [here](https://golang.org/pkg/net/url/#example_URL)

Reply:
Yeah we can follow this practice as well. I just copied the same way as written in line 262 where we are appending SAS in the source URL string to keep things similar. 

```
if len(srcSAS) > 0 {
		sUrl, e := url.Parse(src)
		if e != nil {
			panic(e)
		}
		if len(sUrl.RawQuery) > 0 {
			sUrl.RawQuery += "&" + srcSAS
		} else {
			sUrl.RawQuery = srcSAS
		}
		src = sUrl.String()
	}
```